### PR TITLE
✨ Small workflow updates

### PR DESCRIPTION
**Pull request by asim-shrestha**
Splitting up for a smaller PR

- Update right sidebar to be dynamic
- Add basics of the workflow node logic
- Add frontend to display available workflow nodes: Still need to make it clickable, actually do stuff

### CODE REVIEW

File: docs/development/workflows.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (awtkns) [inline docs/development/workflows.mdx:1]: Nice!

**Review by awtkns — APPROVED** [review]

File: next/public/tools/web.png
Comment (awtkns) [inline next/public/tools/web.png:1]: wonder if FA icons can be used instead

File: platform/reworkd_platform/web/api/workflow/blocks/web/status_check.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    description = "Outputs the status code of a GET request to a URL"
+    image_url = ""
+    input_config: UrlStatusCheckBlockInput
+
+    def run(self) -> BlockIOBase:
+        try:
+            response = requests.get(self.input_config.url)
+            code = response.status_code
+        except RequestException:
+            code = None
```
Comment (awtkns) [inline platform/reworkd_platform/web/api/workflow/blocks/web/status_check.py:28]: might be good to log something?

File: next/public/tools/web.png
Comment (asim-shrestha) [reply next/public/tools/web.png:1]: Not sure we'll have applicable icons for all images in the future

File: platform/reworkd_platform/web/api/workflow/blocks/web/status_check.py (hunk 2 above)
Comment (asim-shrestha) [reply platform/reworkd_platform/web/api/workflow/blocks/web/status_check.py:28]: Yeah we need better logging throughout

**Pull request merged**